jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/deadlines/notifications/impl/NotificationListenerManager.java

```diff
@@ -30,18 +32,59 @@
  *
  */
 public class NotificationListenerManager {
-    
+
+    public static final String KIE_LISTENER_EXCLUDE = "org.kie.jpbm.notification_listeners.exclude";
+
+    public static final String KIE_LISTENER_INCLUDE = "org.kie.jpbm.notification_listeners.include";
+
     private static final Logger logger = LoggerFactory.getLogger(NotificationListenerManager.class);
-    
+
     private static ServiceLoader<NotificationListener> listenersLoaded = ServiceLoader.load(NotificationListener.class);
-    
+
     private static NotificationListenerManager INSTANCE = new NotificationListenerManager();
-    
+
     private List<NotificationListener> listeners = new ArrayList<NotificationListener>();
-    
+
+    private List<String> excludeLists;
+
+    private List<String> includeLists;
+
+    public static List<String> propertyToList(String propertyName) {
+        String name = System.getProperty(propertyName);
+        if (name == null) {
+            return null;
+        }
+        return Arrays.asList(name.split(","));
+    }
+
     private NotificationListenerManager() {
+        init();
+    }
+
+    public void reset() {
+        init();
```

Comment:
Is this method really needed? It's similar to make init public

Reply:
yep... this is extracting a method without changing the constructor and reusing the code for reset which helpfull when testing.